### PR TITLE
[CORE-2000] WorkOrder Update Materials

### PR DESCRIPTION
**Pull request by tiagobastos**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3569,6 +3569,10 @@ Update a draft work order.
         + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional, nullable)
         + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional, nullable)
         + location (Address, optional, nullable)
+        + materials (array, optional, nullable)
```
Comment (WouterSioen) [inline apiary.apib]: Q: I don't think we normally make arrays nullable. If you want to remove all materials, you just pass an empty list of materials, right?

Comment (JordyDeGroeve) [reply apiary.apib]: So this should become an array type that's optional, no?

**Review by JordyDeGroeve — COMMENTED** [review]
Let's adjust the `material` requirements.

File: apiary.apib (hunk 1 above)
Comment (dietrichm) [reply apiary.apib]: Yup, just optional.

**Review by WouterSioen — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Pull request merged**